lib/TPP/ConvertTppToXsmm.cpp

```diff
@@ -72,10 +72,20 @@ struct ConvertTppMatmulOp : public OpRewritePattern<tpp::MatmulOp> {
         rewriter.getContext(), ArrayRef<int64_t>{m, n, k, lda, ldb, ldc});
     xsmm::TernaryKindAttr attr = xsmm::TernaryKindAttr::get(
         matmulOp.getContext(), xsmm::TernaryKind::MATMUL);
-    xsmm::DataTypeAttr dtype =
-        xsmm::DataTypeAttr::get(matmulOp.getContext(), xsmm::DataType::F32);
+    xsmm::DataTypeAttr dtype;
```

Comment:
FYI: I have a change coming-in on this. It is just too much code. I pulled into an helper function. Let's land this as is and I will rebase and clean-up.